lib/tasks/webpack.rake

```diff
@@ -1,7 +1,22 @@
 # frozen_string_literal: true
+
 namespace :webpack do
   desc 'Run development server'
   task :dev do
-    exec('./bin/webpack-dev-server')
+    exec('yarn dev')
   end
+
+  desc "Compile everything under app/javascript into public/packs and generates production assets manifest"
+  task :compile do
+    exec('yarn build')
+  end
+
+  task :check_asset_host do
```

Comment:
I've been testing the CDN functionality and it works perfectly. In fact, in this branch there's room for some cleanup:

- This task `webpack:check_asset_host` is not needed anymore
- Also, `ApplicationHelper.rails_asset_url_tag` and all calls to it are not needed as well.